packages/bot-web-ui/src/stores/main-content-store.js

```diff
@@ -31,6 +33,12 @@ export default class MainContentStore {
     onMount() {
         window.addEventListener('resize', this.setContainerSize);
     }
+    @action.bound
+    cached_active_tab() {
```

Comment:
Please use camelCase for functions e.g.

```suggestion
    getCachedActiveTab() {
```